src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -666,7 +666,7 @@ protected virtual void RequestRun()
         /// </summary>
         protected virtual void NodeModified(NodeModel node)
         {
-
+            HasUnsavedChanges = true;
```

Comment:
What if a workspace overrides this method?  Wouldn't it make more sense to have a private method that does this?